portals/publisher/src/main/webapp/source/src/app/components/GlobalPolicies/Policies/GlobalSpecificComponents/Policies.tsx

```diff
@@ -0,0 +1,610 @@
+                inputResponse.policyMapping.request = request.map((item: any) => ({
+                    ...item,
+                    uuid: uuidv4(),
+                }));
+            }
+            if (response) {
+                inputResponse.policyMapping.response = response.map((item: any) => ({
+                    ...item,
+                    uuid: uuidv4(),
+                }));
+            }
+            if (fault) {
+                inputResponse.policyMapping.fault = fault.map((item: any) => ({
+                    ...item,
+                    uuid: uuidv4(),
+                }));
+            }
+        }  
+        return inputResponse;
+    };
+      
+    /**
+     * Remove UUIDs from the input.
+     * Each Policy operation has a uuid to identify (Since two operations can have same policy ID).
+     * This requires for the UI so removing this is required before sending to backend to overcome backend validation.
+     * @param {any} input - Global Level Policies which has UUIDs for each operations.
+     * @returns {any} - Global Level Policies which does not have UUIDs for each operations.
+     */
+    const removeUUIDs = (input: any) => {
+        const inputWithoutUUIDs: any = cloneDeep(input);
+        if (inputWithoutUUIDs) {
+            const { request, response, fault } = inputWithoutUUIDs;
+            if (request) {
+                inputWithoutUUIDs.request = request.map((item: any) => {
+                    const { uuid, ...rest } = item;
+                    return rest;
+                });
+            }
+            if (response) {
+                inputWithoutUUIDs.response = response.map((item: any) => {
+                    const { uuid, ...rest } = item;
+                    return rest;
+                });
+            }
+            if (fault) {
+                inputWithoutUUIDs.fault = fault.map((item: any) => {
+                    const { uuid, ...rest } = item;
+                    return rest;
+                });
+            }
+        }
+        return inputWithoutUUIDs;
+    };
+
+    /**
+     * If this is the editng page (isCreateNew:False), fetching the data from backend.
+     */
+    const fetchGlobalPolicyByID = () => {
+        setLoading(true);
+        const gatewayPolicyMappingId = String(policyID);
+
+        /**
+         * Backend Call and handle the response.
+         */
+        const promisedPolicy = API.getGatewayPolicyMappingContentByPolicyMappingId(gatewayPolicyMappingId);
+        promisedPolicy
+            .then((response) => {
+                const responseUpdated = assignUUIDs(response.body);      
+                setGlobalLevelPolicies(responseUpdated.policyMapping);
+                setDescription(responseUpdated.description);
+                setName(responseUpdated.displayName);
+                setAppliedGatewayLabels(responseUpdated.appliedGatewayLabels);
+            })
+            .catch((/* error */) => {
+                // console.error(error);
```

Comment:
```suggestion
                
```

Reply:
Addressed the required changes by [fddf57f](https://github.com/wso2/apim-apps/pull/515/commits/fddf57fa7308a53094c808e9bf2d2b40ed71b644) and [0ad490c](https://github.com/wso2/apim-apps/pull/515/commits/0ad490c497b08f999babbe521feaf83d029a6698)